molecule/rbac-plain-provided-debian/molecule.yml

```diff
@@ -219,6 +222,9 @@ provisioner:
           ldap.group.member.attribute.pattern: CN=(.*),OU=rbac,DC=example,DC=com
           ldap.user.object.class: account
 
+          listener.name.client.plain.sasl.jaas.config: org.apache.kafka.common.security.plain.PlainLoginModule required;
```

Comment:
```suggestion
          # interbroker listener need to include user/password in this jaasconfig
          listener.name.client.plain.sasl.jaas.config: org.apache.kafka.common.security.plain.PlainLoginModule required;
```

Reply:
I think add a comment saying what it should look like for the interbroker listener